app/config/errors.php

```diff
@@ -489,6 +489,11 @@
         'description' => 'Project with the requested ID could not be found. Please check the value of the X-Appwrite-Project header to ensure the correct project ID is being used.',
         'code' => 404,
     ],
+    Exception::PROJECT_ALREADY_EXISTS => [
+        'name' => Exception::PROJECT_ALREADY_EXISTS,
+        'description' => 'Project with the requested ID already exists.',
+        'code' => 403,
```

Comment:
We use 409 for others so maybe use the same for this.

Reply:
Yes true but @TorstenDittmann suggested using 403. Which I feel is also relevant in this case 🤔

Reply:
should be 409, I might have remembered the wrong one when I mentioned it :)

Reply:
Okay 👍  updated it